R/cube-residuals.R

```diff
@@ -54,12 +54,8 @@ setMethod('rstandard', 'CrunchCube', function (model) {
              "slice to evaluate.")
     }
 
-    # determine which dimensions are multiple response, and treat those special
-    # TODO: make this specific to multiple response rather than both cat array and MR
-    types <- vapply(dimensions(model), function (dim) dim$references$type, character(1))
-
-    if (any(types == 'subvariable_items')) {
-        return(standardizedMRResiduals(model, types))
+    if (any(grepl("^mr_", getDimTypes(model)))) {
```

Comment:
Stylistically, you could do `any(startsWith(getDimTypes(model), "mr_"))` and it would read nicely.